internal/utils/csi.go

```diff
@@ -181,6 +184,29 @@ func PodsMountDirVolume(kubeletDirPath string) corev1.Volume {
 		},
 	}
 }
+func LogsDirVolume(logHostPath, pluginName string) corev1.Volume {
```

Comment:
Our terminology does not use `pluginName`but `driverName`. Please align

Reply:
We are passing `utils.LogsDirVolume(logHostPath, daemonSet.Name)` not sure whats need to get change

Reply:
You should not use daemonSet.Name as the daemon set is not part of the desired state and there is no guarantee on its value. Please build the name from the driver's name.

This also means that `pluginName` is not the correct terminology

Reply:
It is according to our logic 
```
Volume = <logHostPath>/<pluginName>
Volume Mount = /csi-logs 
Container Arg = /csi-logs/<container-name> 
```

I dont want it to break
Plus `daemonSet.Name` or `deploy.Name` is required, as to make different folders need to get created for daemonsets and deplouments pods

Reply:
@nb-ohad

Reply:
or I can do `Volume = <logHostPath>/<driverNmae>/<pluginName>`